Source/Library/UIColor+Hex.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public extension UIColor {
+    
+    convenience init(hex string: String) {
```

Comment:
Also, I don't think that dependencies are necessary bad by default, I think it is especially fine if they are your own dependencies. But with that said, one should also be apply caution when using them so that you don't end up with a spiderweb.